libkbfs/reporter_kbpki.go

```diff
@@ -230,23 +232,37 @@ func (r *ReporterKBPKI) NotifySyncStatus(ctx context.Context,
 	}
 }
 
+// SuppressNotifications implements the Reporter interface for ReporterKBPKI.
+func (r *ReporterKBPKI) SuppressNotifications(
+	ctx context.Context, nextSuppressIn time.Duration) {
+	r.suppressCh <- nextSuppressIn * 2
```

Comment:
I don't understand why we `*2` here, why not just let the caller set the real value?  Also, why is it called `nextSuppressIn` everywhere?  Isn't this supposed to be the `suppressDuration` or something?

Reply:
The idea is this is like a heartbeat. If caller hasn’t called this RPC for 2*nextSuppressIn, the suppression stops. It needs to be larger than the actual calling interval to count for delays. I guess calling it suppressDuration and pass in the 2*’ed value is clearer. So will do that. Let me know if that’s not what you mean!

Reply:
I just think `nextSuppressIn` sounds like the next suppression isn't supposed to happen for that amount of time, not that it will _be_ suppressed for that amount of time.